cmd/opm/alpha/render-graph/cmd.go

```diff
@@ -67,13 +68,17 @@ $ opm alpha render-graph quay.io/operatorhubio/catalog:latest | \
 				log.Fatal(err)
 			}
 
-			writer := declcfg.NewMermaidWriter(declcfg.WithMinEdgeName(minEdge), declcfg.WithSpecifiedPackageName(specifiedPackageName))
+			writer := declcfg.NewMermaidWriter(
+				declcfg.WithMinEdgeName(minEdge),
+				declcfg.WithSpecifiedPackageName(specifiedPackageName),
+				declcfg.WithV0Semantics(includeV0Semantics))
 			if err := writer.WriteChannels(*cfg, os.Stdout); err != nil {
 				log.Fatal(err)
 			}
 		},
 	}
 	cmd.Flags().StringVar(&minEdge, "minimum-edge", "", "the channel edge to be used as the lower bound of the set of edges composing the upgrade graph; default is to include all edges")
 	cmd.Flags().StringVarP(&specifiedPackageName, "package-name", "p", "", "a specific package name to filter output; default is to include all packages in reference")
+	cmd.Flags().BoolVar(&includeV0Semantics, "include-v0-semantics", false, "whether to indicate OLMv0 semantics in the output; default is to simply represent the upgrade graph")
```

Comment:
My nit is `s/include/use/` for the name of the flag/variable, etc.

"Include" implies multiple $things could be included, all contributing. Whereas "use" has more of a mutual exclusive feel, which I think is more of what we're going for? I also notice that the actual mermaid renderer uses "use" terminology, so this would have better overall alignment as well.